README.md

```diff
@@ -16,6 +16,9 @@ Note: requires MongoDB and RabbitMQ to be running to start correctly.
 
     sudo node index.js
 
+ * http://127.0.0.1:80/ui -- MonoRail Web UI
```

Comment:
@rolandpoulter I double confirm that the default port of `on-http` is 8080, or your browser's default port is set to 8080, so  you could access 127.0.0.1/docs or 127.0.0.1/ui  correctly without explicit 8080 port? I apply this PR's patch and verify again, the port is 8080, if I miss something, please correct me